Updated README, documentation assets, and content (#1028)
liveblocks/liveblocks: README.md

enabling a different facet of collaborative experiences:
[Presence](https://liveblocks.io/docs/products/presence),
[Broadcast](https://liveblocks.io/docs/products/broadcast),

> [Storage](https://liveblocks.io/docs/products/storage), and
> [Comments (private beta)](https://liveblocks.io/docs/products/comments). You can
> decide what products you want to use based on your requirements and
> collaborative experiences you’re looking to build.

Reviewer: ```suggestion
[Storage](https://liveblocks.io/docs/products/storage), and
[Comments (private beta)](https://liveblocks.io/docs/products/comments). You can
decide which products you’d like to use based on the requirements of
the collaborative experiences you’re looking to build.
```
I try to avoid the word "want" in neutral-sounding documentation, it can sound _slightly_ negative as it's used for demanding something.

integrations for specific libraries and frameworks, or create a custom one for
your needs.

- [`@liveblocks/client`](https://liveblocks.io/docs/api-reference/liveblocks-client)
- [`@liveblocks/react`](https://liveblocks.io/docs/api-reference/liveblocks-react)
- [`@liveblocks/redux`](https://liveblocks.io/docs/api-reference/liveblocks-redux)
- [`@liveblocks/zustand`](https://liveblocks.io/docs/api-reference/liveblocks-zustand)
- `@liveblocks/react-comments` (private beta)

### Platform

Liveblocks provides a fully-hosted platform built around a WebSocket edge

> infrastructure that effortlessly scales to millions of users. With the platform,
> you get a lot of powerful tools such as

Reviewer: ```suggestion
infrastructure that effortlessly scales to millions of users. The platform,
equips you with a set of powerful tools such as our
```
